static-site/components/nav/nav-logo.tsx

```diff
@@ -27,11 +27,15 @@ const Logo = (): React.ReactElement => (
 );
 
 const LogoType = (): React.ReactElement => {
-  const rainbowTitle = siteTitle
-    .split("")
-    .map((letter, i) => (
-      <span key={i} style={{ color: `var(--titleColor${i})` }}>{letter}</span>
-    ));
+  const rainbowTitle = siteTitle.split("").map((letter, i) => (
+		<span
+			key={i}
+			style={{ color: `var(--titleColor${i})` }}
+			className={styles.rainbowTitle}
```

Comment:
This is not the right place for this, as we also want the subtitle hidden.

Instead of this, please handle this by either wrapping the returned value (around L44) in a `<div>` that applies this class, or just apply the class to the `<a>` that's wrapping both the title and the subtitle.